src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -484,23 +484,16 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
     winrt::fire_and_forget TermControl::RenderEngineSwapChainChanged()
     {
-        { // lock scope
-            auto terminalLock = _terminal->LockForReading();
-            if (!_initializedTerminal)
-            {
-                return;
-            }
-        }
-
+        // This event is only registered during terminal initialization,

```

Comment:
```suggestion
        // See GH#5203
        // This event is only registered during terminal initialization,
```

Because otherwise I'd forget why this is okay